src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">
+      {person.age ? <p>I am ${person.age}</p> : null}
+    </p>
```

Comment:
Using a `<p>` tag inside another `<p>` tag is not valid HTML. Consider using a `<span>` or directly rendering the text within the outer `<p>` tag.